### ISSUE TITLE
NullPointerException when searching for a printer

### ISSUE THREAD

**Issue by tresf** [qzind/tray#946]
```
Searching for PrintService matching Brother HL-L2380DW series
[ERROR] 2022-03-24T14:21:53,782 @ qz.ws.PrintSocketClient:-1
Problem processing message
java.lang.NullPointerException: null
at qz.printer.PrintServiceMatcher.matchPrinter(Unknown Source) ~[qz-tray.jar:?]
at qz.printer.PrintServiceMatcher.matchPrinter(Unknown Source) ~[qz-tray.jar:?]
at qz.printer.PrintServiceMatcher.findPrinterName(Unknown Source) ~[qz-tray.jar:?]
at qz.ws.PrintSocketClient.processMessage(Unknown Source) ~[qz-tray.jar:?]
at qz.ws.PrintSocketClient.onMessage(Unknown Source) ~[qz-tray.jar:?]
at jdk.internal.reflect.GeneratedMethodAccessor5.invoke(Unknown Source) ~[?:?]
at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
at org.eclipse.jetty.websocket.common.events.annotated.CallableMethod.call(CallableMethod.java:70) ~[qz-tray.jar:?]
at org.eclipse.jetty.websocket.common.events.annotated.OptionalSessionCallableMethod.call(OptionalSessionCallableMethod.java:68) ~[qz-tray.jar:?]
at org.eclipse.jetty.websocket.common.events.JettyAnnotatedEventDriver.lambda$onTextFrame$1(JettyAnnotatedEventDriver.java:270) ~[qz-tray.jar:?]
at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:883) ~[qz-tray.jar:?]
at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1034) ~[qz-tray.jar:?]
at java.lang.Thread.run(Thread.java:829) ~[?:?]
```

**Comment by tresf** [comment]
I believe this is another regression caused by https://github.com/qzind/tray/pull/935.

**Comment by tresf** [comment]
Patched via 67b6448.
